src/components/WalletDropdown/SlideOutMenu.tsx

```diff
@@ -67,7 +67,7 @@ const StyledChevron = styled(ChevronLeft)`
 const BackSection = styled.div`
   position: absolute;
   background-color: ${({ theme }) => theme.backgroundSurface};
-  width: 100%;
+  width: 99%;
```

Comment:
I will come up with something better later today, but I want wallet to look right for bug bash

Reply:
this juts out, it's a real eye sore rn

Reply:
okay, this is jutting out b/c I added a border to the wallet.  The backsection overlays on top of the border